pages/pil/application/views/procedure-data.jsx

```diff
@@ -38,13 +38,17 @@ class ProcedureData extends Component {
                 </dl>
               </div>
               <div className="govuk-grid-column-one-quarter">
-                <Link
-                  page="pil.procedures"
-                  establishment={ establishment.id }
-                  pil={ pil.id }
-                  profile={profile.id}
-                  label={<Snippet>actions.remove</Snippet>}
-                />
+                <ul className="actions">
+                  <li>
+                    <Link
+                      page="pil.procedures"
+                      establishment={ establishment.id }
+                      pil={ pil.id }
+                      profile={profile.id}
+                      label={<Snippet>actions.remove</Snippet>}
+                    />
+                  </li>
+                </ul>
```

Comment:
cool list, bro

Reply:
Originally we had multiple actions per row (edit / remove) 😃

We can fix the styling to not require the lists but I don't think we should do that until we're sure the edit action is definitely not coming back for any of the sections.